torch/distributed/tensor/parallel/style.py

```diff
@@ -39,6 +39,10 @@ class ParallelStyle(ABC):
     """
     The parallel style user wants the module or submodule to be parallelized.
     Users can extend this class to build their own parallel style with customized input/output preparations.
+
+    .. warning::
```

Comment:
Oh yes these are still needed, I'm wondering if there's way to hide those args from the doc

Reply:
Actually given that we already throw deprecation warnings in the doc, I think we should simply remove those APIs from the doc to avoid confusion to new users.

Reply:
No, removing API from docs is now allowed...

Reply:
Hmm removing them from doc will lead to CI failure because we export these are public APIs.. And if we remove the public APIs export, we need to deal with BC break issue. Maybe let't keep the doc here and we can have a different PR to remove doc and make API private later after the conference.